README.md

```diff
@@ -39,6 +39,15 @@ saucectl run --test-env sauce
 
 ![sauce cloud example](assets/sauce_cloud_example.gif)
 
+### In Sauce Cloud with iOS Simulators
+# TODO (change gif file to ios + sauce mode)
```

Comment:
```suggestion
```

Reply:
It's still draft. I left this comment not to forget to create a new gif file.